src/I18n/Translator.php

```diff
@@ -58,7 +58,7 @@ public function translate($key, array $tokensValues = [])
             unset($tokensValues['_context']);
         }
 
-        if (!$tokensValues) {
+        if (empty($tokensValues)) {
```

Comment:
We shouldnt do this.

Reply:
Why not? `$tokenValues` is type hinted as array so no other empty values can sneak in.

Reply:
Exactly,  thats why this not necessary and against conventions. 
Dont use cloaking where var is defined. If someone makes a typo here it will not throw notice.

Reply:
If you make a typo then pslam's `UndefinedVariable` check kicks in, so it's a non-issue.

https://psalm.dev/r/00a573221e

Reply:
Ok. still think its bad to use cloaking to please tools bad reporting, but fair enough then.